src/Common/tests/System/Diagnostics/RemoteExecutorConsoleApp/project.json

```diff
@@ -1,13 +1,13 @@
 {
   "dependencies": {
-    "Microsoft.NETCore.Platforms": "1.0.1-rc3-23910",
-    "System.Console": "4.0.0-rc3-23910",
-    "System.IO": "4.0.10",
-    "System.Reflection": "4.0.0",
-    "System.Runtime": "4.0.20",
-    "System.Threading.Tasks": "4.0.10"
+    "Microsoft.NETCore.Platforms": "1.0.1-rc3-23918",
```

Comment:
These test projects currently expect RID-resolved assets because they behave as standalone apps. We need to change them all to build as NETStandard libraries and change the way they get deployed/run.